Files openal-soft-1.15.1.orig/.CMakeLists.txt.swp and openal-soft-1.15.1/.CMakeLists.txt.swp differ
diff -Nru -x '*~' openal-soft-1.15.1.orig/OpenAL32/Include/alMain.h openal-soft-1.15.1/OpenAL32/Include/alMain.h
--- openal-soft-1.15.1.orig/OpenAL32/Include/alMain.h	2012-12-11 21:35:55.000000000 +0100
+++ openal-soft-1.15.1/OpenAL32/Include/alMain.h	2013-04-11 21:47:41.173462600 +0200
@@ -118,7 +118,7 @@
 
 typedef void *volatile XchgPtr;
 
-#if defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 1))
+#if defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 1)) && !defined(_WIN32)
 typedef ALuint RefCount;
 static __inline RefCount IncrementRef(volatile RefCount *ptr)
 { return __sync_add_and_fetch(ptr, 1); }
